ext/standard/net.c

```diff
@@ -184,6 +193,9 @@ PHP_FUNCTION(net_get_interfaces) {
 	for (p = pAddresses; p; p = p->Next) {
 		zval iface, unicast;
 
+		if ((p->Flags & filter) != filter) {
```

Comment:
It turns out `p->Flags` has a completely different clue on Windows :/ Checking to get a suitable implementation.